src/assets/sass/protocol/utilities/_inline-list.scss

```diff
@@ -0,0 +1,21 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+@import '../includes/lib';
+
+.mzp-u-inline-list {
+    margin: $layout-md 0 0 ($layout-md * -1);
+
+    li {
+        display: inline-block;
```

Comment:
Should this be wrapped in a media query so it defaults to stacked on small screens?

Reply:
I tested it both ways and as long as the links are short I think it works better like this, even on small screens. If it proves to be a problem once we have actual content we can revisit.